### DISCUSSION TITLE
my application not uploding

### DISCUSSION THREAD

**Discussion by callmeshuvam** [community/community#31243] [GitHub Education]
Your application cannot be reviewed until you fix the following:
You appear not to be near any campus location for the school you have selected. If you are a distance learner then your school-provided academic affiliation documentation must state so.

it's show's when i'm uploding 
can anyone tell me what i do please h
![WhatsApp Image 2022-08-29 at 9 50 47 PM](https://user-images.githubusercontent.com/112412569/187247945-c92ee68f-90a6-4e89-94c0-1f49a80ae554.jpeg)
ow to solve this problem and get Github student  pack

**Comment by KasperiP** [comment]
As it says it cannot process your application because your location (GPS-location) is not near the campus you are trying to use for the application. Make sure you have GPS enabled and you are close enough the campus OR as it says provide school-provided academic affiliation documentation that proofs that you are distance learner (remote, not actively studying on campus).

**Comment by leothehero** [comment]
I am having the same issue. I am doing the application through my computer; all location services are active; I am physically on my campus, and I receive this error. On top of the rediculous design that requires a camera (and likely only through a bug, allows you to drag and drop jpgs to upload them) to take a picture of a document that may be digital only, and the fact that on mobile it does not allow you to switch the camera (despite having a UI for doing so), this system that doesn't even allow a human to correct for computer error baffles me. How can you supply proof of residency or being a distance learner if the system refuses to let you submit anything until proven???
